MidCamp/midcamp
# Remove workflow (ALT)

## Description

This is a lighter alternative to #311, which requires Form Mode Manager and ultimately changes node/edit routes for different forms.

I cherry-picked most of this from that branch, however once I set Topic nodes' default state to unpublished, it achieved the desired effect.  

## Testing

https://nginx-midcamp-org-feature-workflow-shmerkflow-2-the-reckoning.us.amazee.io/

- Log in as an authenticated user.  
- Create a new Topic node at https://nginx-midcamp-org-feature-workflow-shmerkflow-2-the-reckoning.us.amazee.io/node/add/topic
- Observe you cannot directly publish this node
